src/Sarif.Sarifer/Properties/VersionConstants.cs

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.CodeAnalysis.Sarif.Sarifer
+{
+    public static class VersionConstants
```

Comment:
>VersionConstants [](start = 24, length = 16)

The `VersionConstants` files are generated in each project. You won't have to edit them.